data/i18n/en/en.toml

```diff
@@ -49,26 +49,20 @@ other = "CVE ID"
 [cve_issue_url]
 other = "CVE GitHub Issue URL"
 
-[cve_json_external_url]
-other = "external_url"
-
-[cve_json_id]
-other = "id"
-
-[cve_json_summary]
-other = "summary"
-
-[cve_json_url]
-other = "url"
-
 [cve_summary]
 other = "Issue Summary"
 
 [cve_table]
 other = "Official Kubernetes CVE List"
 
-[cve_url]
-other = "CVE URL"
+[cve_table_date_before]
+other = "(last updated: "
```

Comment:
This is a bit unusual
I'll look at how you are constructing the string

Reply:
I was thinking on the contrary that it was how we usually do these things, but we can do whatever you think is better.
https://github.com/kubernetes/website/blob/206231dcac5fbcce000bbc21a01a6a9806b836d4/data/i18n/en/en.toml#L275-L279

Reply:
I prefer to have the localized string be a _format string_:
```toml
[cve_table_date_with_label]
other = "(last updated: %s)"
```
and then use a string formatter to include the localized date. That's equivalent, but the placeholder makes it easier for localization teams to understand.

Reply:
I'll update this in the follow-up PR https://github.com/kubernetes/website/pull/39513 :)